source-academy/frontend
# Implement participant indicator for sessions

### Description

Adds cursor display for other users in the same session and fixed related bugs.
This PR is associated with https://github.com/source-academy/sharedb-ace-backend/pull/26 and https://github.com/source-academy/sharedb-ace/pull/26/

### Type of change


### How to test

Multiple users join the same session. Upon entering, cursors of other users should automatically show. Each cursor whenever updated will move to its new position and show a label containing the name of the user associated with it ("undefined" if no name is detected).

To test the `sharedb-ace` package locally, run `yarn build` then `yarn link` on the `sharedb-ace` repository, then run `yarn link @sourceacademy/sharedb-ace` on the frontend repository.

### Checklist
